guacamole/src/main/webapp/app/navigation/services/userPageService.js

```diff
@@ -57,12 +57,24 @@ angular.module('navigation').factory('userPageService', ['$injector',
      *     A map of all root connection groups visible to the current user,
      *     where each key is the identifier of the corresponding data source.
      *
+     * @param {Object.<String, PermissionSet>} permissions
+     *     A map of all permissions granted to the current user, where each
+     *     key is the identifier of the corresponding data source.
+     *
+     * @param {Object.<String,
```

Comment:
Looks like some copypasta snuck in here.

Reply:
Ha.  I started to type it myself, then copied it from the function that calls this one and forgot to remove the copy I started to type.  Like any good pasta, I've cleaned it up.